lib/atlassian-jwt-authentication/middleware.rb

```diff
@@ -8,12 +8,18 @@ class VerifyJwtToken
       JWT_ACCOUNT_ID = "#{PREFIX}.account_id".freeze
       JWT_QSH_VERIFIED = "#{PREFIX}.qsh_verified".freeze
 
-      def initialize(app, addon_key)
+      def initialize(app, options)
         @app = app
-        @addon_key = addon_key
+        @addon_key = options[:addon_key]
+        @condition = options[:condition]
```

Comment:
I'd prefer `:if` to be honest (to follow Rails), but it's fine and we're in hurry to fix it.

Reply:
Good idea.